### PR TITLE
refactor(egen): template fixes, adds clean up steps 

### PR DESCRIPTION
**Pull request by UBhavani**
This PR consists of the following updates:

    1. Refactors the notebook according to the notebook template (removes boilerplate elements, renames REGION with LOCATION and re-structures the sections).
    2. Removes the usage of future tense(will) according to the guidelines.
    3. Adds model deletion steps in clean up section.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 1]
```diff
@@ -133,186 +138,181 @@
...
+        "id": "de998a3953c9"
       },
-      "outputs": [],
       "source": [
-        "# Install required packages.\n",
-        "! pip3 install --quiet --upgrade google-cloud-aiplatform \\\n",
-        "                                 google-cloud-bigquery"
+        "### Objective\n",
+        "\n",
+        "In this notebook, you learn to use the `Vertex AI Model Monitoring` service to detect feature skew and drift in the input predict requests for XGBoost models.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:155]: remove back ticks

- Vertex AI Model Monitoring.  (It's a product/service name)

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 2]
```diff
@@ -133,186 +138,181 @@
...
+        "\n",
+        "This tutorial uses the following Google Cloud ML services:\n",
+        "\n",
+        "- `Vertex AI Model Monitoring`\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:159]: remove back ticks

- Vertex AI Model Monitoring

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 3]
```diff
@@ -133,186 +138,181 @@
...
+        "- `Vertex AI Prediction`\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:160]: remove back ticks

- Vertex AI Prediction

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 4]
```diff
@@ -133,186 +138,181 @@
...
+        "- `Vertex AI Model` resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:161]: remove back ticks & lower case "model"

- Vertex AI model

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 5]
```diff
@@ -133,186 +138,181 @@
...
+        "- `Vertex AI Endpoint` resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:162]: remove back ticks & lower case "endpoint"

- Vertex AI endpoint

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 6]
```diff
@@ -492,13 +493,13 @@
...
         "- Set the variable `DEPLOY_COMPUTE` to configure your compute resources for prediction.\n",
-        " - `machine type`\n",
+        " - Set a `machine type`:\n",
         "     - `n1-standard`: 3.75GB of memory per vCPU\n",
         "     - `n1-highmem`: 6.5GB of memory per vCPU\n",
         "     - `n1-highcpu`: 0.9 GB of memory per vCPU\n",
         " - `vCPUs`: number of \\[2, 4, 8, 16, 32, 64, 96 \\]\n",
         "\n",
-        "*Note: You may also use n2 and e2 machine types for training and deployment, but they do not support GPUs*."
+        "**Note**: You may also use n2 and e2 machine types for training and deployment, but they do not support GPUs."
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:502]: change "do not" to "don't"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 7]
```diff
@@ -530,26 +531,26 @@
...
         "\n",
-        "Vertex AI Model Monitoring is supported for AutoML tabular models and custom tabular models. You can monitor for skew and drift detection of the features in the inbound prediction requests or skew and drift detection of the feature attributions (Explainable AI) in the outbound prediction response -- that is, the distribution of the attributions on how they contributed to the output (predictions).\n",
+        "Vertex AI Model Monitoring is supported for AutoML tabular models and custom tabular models. You can monitor for skewness and drift detection of the features in the inbound prediction requests or the feature attributions (Explainable AI) in the outbound prediction response. In other words, you monitor the distribution of the attributions that quantify feature contributions to the output (predictions).\n",
         "\n",
         "The following are the basic steps to enable model monitoring:\n",
         "\n",
-        "1. Deploy a `Vertex AI` AutoML or custom tabular model to an `Vertex AI Endpoint`.\n",
+        "1. Deploy a Vertex AI AutoML or custom tabular model to a Vertex AI Endpoint.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:538]: lower case "Vertex AI endpoint"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 8]
```diff
@@ -530,26 +531,26 @@
...
         "2. Configure a model monitoring specification.\n",
-        "3. Upload the model monitoring specification to the `Vertex AI Endpoint`.\n",
-        "4. Upload or automatic generation of the `input schema` for parsing.\n",
-        "5. For feature skew detection, upload the training data for automatic generation of the feature distribution.\n",
-        "6. For feature attributions, upload corresponding `Vertex AI Explainability` specification.\n",
+        "3. Upload the model monitoring specification to the Vertex AI Endpoint.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:540]: lower case "Vertex AI endpoint"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 9]
```diff
@@ -1138,13 +1135,13 @@
       "source": [
         "### Drift detection during monitoring\n",
         "\n",
-        "The feature input drift detection will occur at the next monitoring interval. In this tutorial, you set the monitoring interval to one hour. So, in about an hour your monitoring job will go from `OFFLINE` to `RUNNING`. While running, it will analyze the logged sampled tables from the predictions during this interval and compare them to the previous monitoring interva distribution.\n",
+        "The feature input drift detection will occur at the next monitoring interval. In this tutorial, you set the monitoring interval to one hour. So, in about an hour your monitoring job goes from **OFFLINE** to **RUNNING**. While running, it analyzes the logged sampled tables from the predictions during this interval and compares them to the previous monitoring interva distribution.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1138]: remove "will" and change "occur" to "occurs":

- The feature input drift detection occurs at the next monitoring interval.

Typo:

- previous monitoring interva distribution.\n",

Should be:

- previous monitoring interval distribution.\n",

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 10]
```diff
@@ -1217,62 +1216,1261 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lA32H1oKGgpf"
+      },
+      "source": [
+        "## Overview\n",
         "\n",
-        "Otherwise, you can delete the individual resources you created in this tutorial."
+        "This tutorial demonstrates how to use `Vertex AI Model Monitoring` for XGBoost models.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1310]: remove back ticks:

- This tutorial demonstrates how to use Vertex AI Model Monitoring for XGBoost models.

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 11]
```diff
@@ -1217,62 +1216,1261 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "de998a3953c9"
+      },
+      "source": [
+        "### Objective\n",
+        "\n",
+        "In this notebook, you learn to use the `Vertex AI Model Monitoring` service to detect feature skew and drift in the input predict requests for XGBoost models.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1323]: Remove back ticks:

- Vertex AI Model Monitoring

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 12]
```diff
@@ -1217,62 +1216,1261 @@
...
+        "\n",
+        "This tutorial uses the following Google Cloud ML services:\n",
+        "\n",
+        "- `Vertex AI Model Monitoring`\n",
+        "- `Vertex AI Prediction`\n",
+        "- `Vertex AI Model` resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1329]: remove back ticks and make "model" lower case:

- Vertex AI model

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 13]
```diff
@@ -1217,62 +1216,1261 @@
...
+        "id": "de998a3953c9"
+      },
+      "source": [
+        "### Objective\n",
+        "\n",
+        "In this notebook, you learn to use the `Vertex AI Model Monitoring` service to detect feature skew and drift in the input predict requests for XGBoost models.\n",
+        "\n",
+        "This tutorial uses the following Google Cloud ML services:\n",
+        "\n",
+        "- `Vertex AI Model Monitoring`\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1327]: remove back ticks

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 14]
```diff
@@ -1217,62 +1216,1261 @@
...
+        "- `Vertex AI Prediction`\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1328]: remove back ticks

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 15]
```diff
@@ -1217,62 +1216,1261 @@
...
+        "- `Vertex AI Model` resource\n",
+        "- `Vertex AI Endpoint` resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1330]: remove back ticks and make "endpoint" lower case:

Vertex AI endpoint

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 16]
```diff
@@ -1217,62 +1216,1261 @@
...
+      "source": [
+        "### Costs \n",
+        "\n",
+        "This tutorial uses billable components of Google Cloud:\n",
+        "\n",
+        "* Vertex AI\n",
+        "* BigQuery\n",
+        "* Cloud Storage\n",
+        "\n",
+        "Learn about [Vertext AI\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1371]: put on one line:

"Learn about [Vertext AI\pricing](https://cloud.google.com/vertex-ai/pricing) and [Cloud Storage\n", 

Want to make sure the link works. Do the same for [Cloud Storage... AND [Pricing... (now in line 1373). Don't want to break up the link text and the corresponding URL.

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 17]
```diff
@@ -1217,62 +1216,1261 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8RJ3_20etd31"
+      },
+      "source": [
+        "### Notes about service account and permission\n",
+        "\n",
+        "**By default no configuration is required**, if you run into any permission related issue, please make sure the service accounts above have the required roles:\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1535]: remove "please"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 18]
```diff
@@ -1217,62 +1216,1261 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "container:training,prediction"
+      },
+      "source": [
+        "#### Set pre-built containers\n",
+        "\n",
+        "Set the pre-built Docker container image for prediction.\n",
+        "\n",
+        "For the latest list, see [Pre-built containers for prediction](https://cloud.google.com/ai-platform-unified/docs/predictions/pre-built-containers)."
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1674]: The current URL involves a redirect. Change the ai-platform URL to:

- https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 19]
```diff
@@ -1217,62 +1216,1261 @@
...
+      "source": [
+        "## Introduction to Vertex AI Model Monitoring\n",
+        "\n",
+        "Vertex AI Model Monitoring is supported for AutoML tabular models and custom tabular models. You can monitor for skewness and drift detection of the features in the inbound prediction requests or the feature attributions (Explainable AI) in the outbound prediction response. In other words, you monitor the distribution of the attributions that quantify feature contributions to the output (predictions).\n",
+        "\n",
+        "The following are the basic steps to enable model monitoring:\n",
+        "\n",
+        "1. Deploy a Vertex AI AutoML or custom tabular model to a Vertex AI Endpoint.\n",
+        "2. Configure a model monitoring specification.\n",
+        "3. Upload the model monitoring specification to the Vertex AI Endpoint.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1749]: lower case:

Vertex AI endpoint

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 20]
```diff
@@ -1217,62 +1216,1261 @@
...
+        "4. Upload schema or use automatic generation of the *input schema* for parsing.\n",
+        "5. For feature skewness detection, upload the training data. This enables automatic generation of the feature distributions.\n",
+        "6. For feature attributions, upload corresponding *Vertex Explainable AI* specification.\n",
+        "\n",
+        "Once configured, you can enable/disable monitoring, change alerts and update the model monitoring configuration. \n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:1754]: change "enable/disable" to "enable or disable"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 21]
```diff
@@ -1217,62 +1216,1261 @@
...
+        "\n",
+        "An email notification is sent to the email address in the alerting configuration, notifying that the model monitoring job is now enabled.\n",
+        "\n",
+        "The contents of the email appear as below:\n",
+        "\n",
+        "<blockquote>\n",
+        "Hello Vertex AI Customer,\n",
+        "\n",
+        "You are receiving this mail because you are using the Vertex AI Model Monitoring service.\n",
+        "This mail is to inform you that we received your request to set up drift or skew detection for the Prediction Endpoint listed below. Starting from now, incoming prediction requests will be sampled and logged for analysis.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:2131]: lower case for "prediction endpoint"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 22]
```diff
@@ -1217,62 +1216,1261 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5184c68e4c99"
+      },
+      "source": [
+        "### Logging sampled requests\n",
+        "\n",
+        "On the next monitoring interval, the sampled predictions are then copied over to the BigQuery logging table. Once the entries are in the BigQuery table, the monitoring service will analyze the sampled data.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:2250]: remove "will" and change "analyze" to "analyzes"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 23]
```diff
@@ -1217,62 +1216,1261 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "683ed0ba4ccd"
+      },
+      "source": [
+        "### Drift detection during monitoring\n",
+        "\n",
+        "The feature input drift detection will occur at the next monitoring interval. In this tutorial, you set the monitoring interval to one hour. So, in about an hour your monitoring job goes from **OFFLINE** to **RUNNING**. While running, it analyzes the logged sampled tables from the predictions during this interval and compares them to the previous monitoring interva distribution.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:2347]: correct typo: "monitoring interva" should be "monitoring interval"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb [hunk 24]
```diff
@@ -1217,62 +1216,1261 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5184c68e4c99"
+      },
+      "source": [
+        "### Logging sampled requests\n",
+        "\n",
+        "On the next monitoring interval, the sampled predictions are then copied over to the BigQuery logging table. Once the entries are in the BigQuery table, the monitoring service will analyze the sampled data.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb:2376]: remove "will" and change "analyze" to "analyzes"

**Review by kittyabs — COMMENTED** [review]
suggested edits and typo corrections

**Review by katiemn — APPROVED** [review]

**Pull request merged**